ui/app/templates/workspace/projects/index.hbs

```diff
@@ -42,12 +42,7 @@
   </Card>
   {{else}}
   <EmptyState>
-    <p>There are no projects to display yet</p>
-    <p>To create your first project, create a new application by running
-      <CopyableCode @ref="empty-init" @inline="true">
-        <code id="empty-init">waypoint init</code>
-      </CopyableCode>
-      from the CLI</p>
+    <p>There are no projects to display yet. Create your first project <LinkTo @route="workspace.projects.new">here</LinkTo>.</p>
```

Comment:
I’d be inclined to make the whole phrase the link, rather than just `here`, as per as [WCAG Success Criterion 2.4.9 Link Purpose (Link Only)](https://www.w3.org/TR/WCAG21/#link-purpose-link-only). Link-in-context is meets the A criteria, but link-only gets us all the way to AAA. It becomes clear why link-only gets more points when you brink up the “Links” list in VoiceOver’s rotor:

<img width="579" alt="CleanShot 2021-09-13 at 11 39 05@2x" src="https://user-images.githubusercontent.com/34030/133061489-cbf8e3df-afec-4339-8295-b4be6403d079.png">